src/src/PluginStructs/P020_data_struct.cpp

```diff
@@ -464,6 +469,7 @@ bool P020_Task::handleP1Char(char ch) {
     # endif // ifndef BUILD_NO_DEBUG
     state = ParserState::WAITING; // reset
   }
+  ch &= 0x7F;                     // Strip off occasional 8th bit for now
```

Comment:
Why do this on ser2net?
On the P1 plugin I can understand it, but why on ser2net?

Reply:
This method `handleP1Char()` is _only_ called when configured as P1 receiver, and now that the features of P044 (predefined configured for P1 only) have been merged with P020, the code all lives in the P020 sources 😉